### PR TITLE
Release 7.0.0

### PR DESCRIPTION
**Pull request by root-expert**
Weird Changelog changes?

Probably blocked by:
* #481 

Closes: #478

### CODE REVIEW

**Comment by bastelfreak** [comment]
can you please rebase?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,13 +4,59 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v7.0.0](https://github.com/voxpupuli/puppet-rundeck/tree/v7.0.0) (2022-01-14)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-rundeck/compare/v6.0.0...v7.0.0)
+
+recent rundeck versions didn't work properly on CentOS 6. This release does not support CentOS 6 anymore!
```
Comment (root-expert) [inline CHANGELOG.md:11]: Where does this come from?

Comment (root-expert) [reply CHANGELOG.md:11]: Ah https://github.com/voxpupuli/puppet-rundeck/issues/456

File: CHANGELOG.md [hunk 2]
```diff
@@ -357,7 +406,26 @@ These should not affect the functionality of the module.
 - should contain JettyCachingLdapLoginModule and be sufficient [\#163](https://github.com/voxpupuli/puppet-rundeck/pull/163) ([dalisch](https://github.com/dalisch))
 - disable H2 database logging in log4j.properties [\#161](https://github.com/voxpupuli/puppet-rundeck/pull/161) ([dalisch](https://github.com/dalisch))
 - Make ssh\_keypath and projects\_dir parameters of the project defined type [\#153](https://github.com/voxpupuli/puppet-rundeck/pull/153) ([jyaworski](https://github.com/jyaworski))
+- updating metadata and changelog for 2.1.0 release [\#152](https://github.com/voxpupuli/puppet-rundeck/pull/152) ([liamjbennett](https://github.com/liamjbennett))
```
Comment (smortex) [inline CHANGELOG.md:409]: This is now appearing bellow 2.2.0 :thinking:

File: CHANGELOG.md [hunk 3]
```diff
@@ -21,14 +66,13 @@ These should not affect the functionality of the module.
...
 **Fixed bugs:**
 
-- Support for Rundeck 3.1 [\#419](https://github.com/voxpupuli/puppet-rundeck/issues/419)
+- Rundeck 3.1 does not require rundeck-config package anymore [\#419](https://github.com/voxpupuli/puppet-rundeck/issues/419)
 - groupdel: cannot remove the primary group of user 'rundeck' [\#199](https://github.com/voxpupuli/puppet-rundeck/issues/199)
 
 **Merged pull requests:**
 
-- Token duration must be a string [\#424](https://github.com/voxpupuli/puppet-rundeck/pull/424) ([danifr](https://github.com/danifr))
+-  Fix tests for \#413  [\#424](https://github.com/voxpupuli/puppet-rundeck/pull/424) ([danifr](https://github.com/danifr))
```
Comment (smortex) [inline CHANGELOG.md:74]: Previous label was more insightful for end-users.

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,13 +4,58 @@ All notable changes to this project will be documented in this file.
...
+- Clean up acceptance spec helper [\#429](https://github.com/voxpupuli/puppet-rundeck/pull/429) ([ekohl](https://github.com/ekohl))
+
 ## [v6.0.0](https://github.com/voxpupuli/puppet-rundeck/tree/v6.0.0) (2019-09-29)
 
 [Full Changelog](https://github.com/voxpupuli/puppet-rundeck/compare/v5.4.0...v6.0.0)
 
 **Breaking changes:**
 
-- Rundeck 3.1 does not require rundeck-config package anymore [\#422](https://github.com/voxpupuli/puppet-rundeck/pull/422) ([danifr](https://github.com/danifr))
+- Fix for \#419 + update test to use latest version [\#422](https://github.com/voxpupuli/puppet-rundeck/pull/422) ([danifr](https://github.com/danifr))
```
Comment (smortex) [inline CHANGELOG.md:58]: Same

**Review by smortex — COMMENTED** [review]
Some weird changes… Also I think some PR titles can be adjusted to better indicate changes without needing the user to click the links.

**Review by coolbluemelvin — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (bastelfreak) [reply CHANGELOG.md:74]: the generator had multithreading issues in the past. the new title is the one also set for the PR. I think it's fine now.

**Pull request merged**